charter.html

```diff
@@ -378,6 +384,10 @@ <h3 id="external-coordination">External Organizations</h3>
 		  <dl>
 			<dt><a href="https://www.ietf.org">IETF</a></dt>
 			<dd>A number of IETF working groups are likely venues for standardization of protocol components that advertising features depend on and research groups are investigating issues that will feed into the designs this group will consider. The main group we will need to coordinate with is the <a href="https://datatracker.ietf.org/wg/ppm/">Privacy Preserving Measurement</a> group</a>
+      <dt><a href="https://www.iab.com/">Interactive Advertising Bureau (IAB)</a></dt>
+      <dd>The IAB works with the media and marketing industries to promotes the interactive advertising industry and develops technical standards and solutions.</dd>
+      <dt><a href="https://iabeurope.eu/">IAB Europe</a></dt>
+      <dd>The IAB Europe is an European-level association for the digital marketing and advertising ecosystem.</dd>
 		  </dl>
```

Comment:
In my experience IAB Tech Lab (which works with but is distinct from both IAB and IAB Europe) has been a valuable external organization in this effort.
```suggestion
      <dt><a href="https://iabtechlab.com/">IAB Tech Lab</a></dt>
      <dd>The IAB Tech Lab produces technical specifications, standards, and software widely used in the digital marketing and advertising ecosystem.</dd>
      </dl>
```

Reply:
I think this is a reasonable change. The IAB Tech Lab identifies as a standard setting organization which is more relevant. We have invited their participation in the past and would almost certainly do so in the future.